### PR TITLE
Add COE x Yuki Aim 2024 1v1 Tournament article

### PR DESCRIPTION
**Pull request by ThePooN**
Added the wiki page for the COE x Yuki Aim 2024 1v1 tournament.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/COE/COE_2024/en.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration and online qualifiers | 2024-07-16/2024-07-29 |
+| Qualifier results reveal & Finals mappool showcase | 2024-07-31 (13:00 UTC+2) |
+| Finals | 2024-08-02/2024-08-03 |
+
+## Prizes
+
+The COE x Yuki Aim 2024 1v1 Tournament offered a total cash prize pool of 3350€, keyboards provided by [Wooting](https://wooting.io/) and several merch from Yuki Aim, including stickers, posters, glasspad / clothpad
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/COE/COE_2024/en.md:15]:
```suggestion
The COE x Yuki Aim 2024 1v1 Tournament offered a total cash prize pool of 3350€, keyboards provided by [Wooting](https://wooting.io/) and several merch from [Yuki Aim](https://yuki.gg/), including stickers, posters, glasspad / clothpad
```

File: wiki/Tournaments/COE/COE_2024/en.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+![COE x Yuki Aim 2024 1v1 Tournament winner badge](img/badge.png "COE x Yuki Aim 2024 1v1 Tournament winner badge")
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Admins | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815), ::{ flag=DE }:: [Meyer](https://osu.ppy.sh/users/5452367), ::{ flag=DE }:: [TheHunter1](https://osu.ppy.sh/users/6496016), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) |
+| Mappool selector | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815), ::{ flag=PL }:: [Bartek22830](https://osu.ppy.sh/users/6404027), ::{ flag=HU }:: [Himada](https://osu.ppy.sh/users/10959366), ::{ flag=RO }:: [nanoya](https://osu.ppy.sh/users/12366071), ::{ flag=DE }:: [himejoshi](https://osu.ppy.sh/users/6754508) |
+| Mapper | ::{ flag=HK }:: [Kurashina Asuka](https://osu.ppy.sh/users/7476493), ::{ flag=SK }:: [Atipir](https://osu.ppy.sh/users/8991722), ::{ flag=UK }:: [DeviousPanda](https://osu.ppy.sh/users/4966334), ::{ flag=EE }:: [iljaaz](https://osu.ppy.sh/users/8501291), ::{ flag=AT }:: [Seamob](https://osu.ppy.sh/users/10840806) |
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/COE/COE_2024/en.md:34]:
```suggestion
| Mapper | ::{ flag=HK }:: [Kurashina Asuka](https://osu.ppy.sh/users/7476493), ::{ flag=SK }:: [Atipir](https://osu.ppy.sh/users/8991722), ::{ flag=GB }:: [DeviousPanda](https://osu.ppy.sh/users/4966334), ::{ flag=EE }:: [iljaaz](https://osu.ppy.sh/users/8501291), ::{ flag=AT }:: [Seamob](https://osu.ppy.sh/users/10840806) |
```

File: wiki/Tournaments/COE/COE_2024/en.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+| Commentator | ::{ flag=UK }:: [Bubbleman](https://osu.ppy.sh/users/5182050), ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251),  ::{ flag=GB }:: [Doomsday](https://osu.ppy.sh/users/18983), ::{ flag=NO }:: [NINERIK](https://osu.ppy.sh/users/10549880), ::{ flag=UK }:: [Malteser](https://osu.ppy.sh/users/5218178), ::{ flag=US }:: [n3rdiness](https://osu.ppy.sh/users/16537481), ::{ flag=DE }:: [himejoshi](https://osu.ppy.sh/users/6754508), ::{ flag=UK }:: [SadShiba](https://osu.ppy.sh/users/10747626), ::{ flag=IT }:: [Spazza17](https://osu.ppy.sh/users/3516241), ::{ flag=MK }:: [sst3w](https://osu.ppy.sh/users/13337202) |
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/COE/COE_2024/en.md:35]:
```suggestion
| Commentator | ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050), ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251),  ::{ flag=GB }:: [Doomsday](https://osu.ppy.sh/users/18983), ::{ flag=NO }:: [NINERIK](https://osu.ppy.sh/users/10549880), ::{ flag=GB }:: [Malteser](https://osu.ppy.sh/users/5218178), ::{ flag=US }:: [n3rdiness](https://osu.ppy.sh/users/16537481), ::{ flag=DE }:: [himejoshi](https://osu.ppy.sh/users/6754508), ::{ flag=GB }:: [SadShiba](https://osu.ppy.sh/users/10747626), ::{ flag=IT }:: [Spazza17](https://osu.ppy.sh/users/3516241), ::{ flag=MK }:: [sst3w](https://osu.ppy.sh/users/13337202) |
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/COE/COE_2024/en.md (hunk 1 above)
Comment (iSlodinxOsu) [reply wiki/Tournaments/COE/COE_2024/en.md:15]: I guess this should be fine to link their website aswell?

**Comment by Joehuu** [comment]
Needs to be added to tournament index.

File: wiki/Tournaments/COE/en.md [hunk 4]
```diff
@@ -4,3 +4,4 @@ Index page for the COE tournament series.
 
 - [COE 1v1 Tournament 2022](COE_2022)
 - [COE 1v1 Tournament 2023](COE_2023)
+- [COE x Yuki Aim 2024 1v1 Tournament 2024](COE_2024)
```
Comment (Joehuu) [inline wiki/Tournaments/COE/en.md:7]:
```suggestion
- [COE x Yuki Aim 2024 1v1 Tournament](COE_2024)
```

**Comment by ThePooN** [comment]
Regarding the badges and banners, I rebased this MR and edited the original commits in order not to inflate the repo, given it's not using Git LFS.

File: wiki/Tournaments/COE/COE_2024/en.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+The players then alternate picking the map. If they reach a tie after playing 8 maps (or 10 during Grand Final or the third place match), the tiebreaker will be picked automatically.\
+Between each map there is a short break of a few seconds during which the next map must be picked.
+
+Once any player reaches 5 map wins (or 6 map wins during Grand Final or the third place match), the match is over and the winner advances to the next match.
+
+Rounds with disconnects within 30 seconds or 25% of the beatmap length, whichever happens first, can be replayed, at the discretion of the referee, as long as the problem was clearly communicated in time.
+
+Should a map end in a draw the beatmap will be replayed, before moving on to the next pick.
+
+### 5\. Code of Conduc
```
Comment (Joehuu) [inline wiki/Tournaments/COE/COE_2024/en.md:260]:
```suggestion
### 5\. Code of Conduct
```

**Review by Joehuu — COMMENTED** [review]
Not gonna nitpick the ruleset as much. Seems fine.

**Review by Joehuu — APPROVED** [review]

**Comment by ThePooN** [comment]
Thanks @iSlodinxOsu and @Joehuu for the reviews and merge.

IMHO, I think some improvements could be done to the scripts and workflows to facilitate this process. Had no idea about the indexes, getting dates to match and properly formatted on every page was a nightmare, could add checks for image dimensions. I had actually bothered to set up and run the scripts and feel a little betrayed :stuck_out_tongue:

**Pull request merged**